docker-compose.yml

```diff
@@ -43,6 +43,7 @@ services:
   #  - Built from "server/Dockerfile"
   # ---------------------------------
   server:
+    platform: linux/amd64
```

Comment:
This is needed because the new Macs use Apple Silicon chips, which are based on the ARM architecture, and the OpenDP library is not built to run on that.  I had that happen last summer, and I updated docker-compose-cypressdev.yml, but I forgot to update docker-compose.yml. 
I'm assuming you have a new Mac too and that's why you had the problem :)